src/CommandingOfficers/AW1/BH/Sturm.java

```diff
@@ -0,0 +1,174 @@
+package CommandingOfficers.AW1.BH;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import CommandingOfficers.*;
+import CommandingOfficers.AW1.AW1Commander;
+import Engine.GameScenario;
+import Engine.XYCoord;
+import Engine.Combat.DamagePopup;
+import Engine.Combat.StrikeParams;
+import Engine.Combat.StrikeParams.BattleParams;
+import Engine.GameEvents.GameEvent;
+import Engine.GameEvents.GameEventQueue;
+import Engine.UnitMods.DamageMultiplierDefense;
+import Engine.UnitMods.DamageMultiplierOffense;
+import Engine.UnitMods.UnitModifier;
+import UI.UIUtils;
+import Units.UnitContext;
+import Units.MoveTypes.MoveType;
+import Terrain.MapMaster;
+import Terrain.TerrainType;
+import Terrain.Environment.Weathers;
+import Terrain.GameMap;
+
+public class Sturm extends AW1Commander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Sturm_1", UIUtils.SourceGames.AW1, UIUtils.BH);
+      infoPages.add(new InfoPage(
+            "Sturm (AW1)\n"
+          + "A riddle within a shadow, revealing nothing.\n"
+          + "???\n"
```

Comment:
Are we just tossing question marks in here for appearance's sake and then explaining everything anyway?

Reply:
Hey, man. That's the original description. I dunno what you want from me.